lametro/templates/partials/past_event_day.html

```diff
@@ -35,11 +35,11 @@ <h5 class="text-default">
     </div>
 
     <div class="col-xs-3">
-        {% for media in event.media %}
-        <p>
+        {% for media in event.media|dictsortreversed:'label' %}
```

Comment:
can you add [a django template comment](https://docs.djangoproject.com/en/2.1/topics/templates/#comments) here, explaining this sort?